src/runtime/plugin.client.ts

```diff
@@ -30,11 +31,13 @@ export default defineNuxtPlugin((nuxtApp) => {
   const runtimeConfig = useRuntimeConfig()
   const hawkConfig = runtimeConfig.public.hawk as HawkModuleConfig
 
-  const hawkInstance = new HawkCatcher({
+  const hawkInstance = new HawkCatcher(Object.assign({}, hawkConfig.catcherOptions, {
     token: hawkConfig.token,
     vue: nuxtApp.vueApp,
     release: getReleaseId(),
-  })
+  }, {
+    beforeSend,
+  }))
```

Comment:
is object assign safe for `hawkConfig.catcherOptions`, as i know this is not deep cloning, maybe we should assert `HawkCatcher`  separately?

Reply:
we don't need deep merging there, just overriding values

Reply:
and `catcherOptions` are assigning to the empty object, so there will be no overrides at all